db/models.py

```diff
@@ -12,5 +12,38 @@ class Actor(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
 
+    def __str__(self) -> tuple[str, str]:
+        return self.first_name, self.last_name
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actor")
+    genres = models.ManyToManyField(Genre, related_name="genre")
```

Comment:
Related names are picked up incorrectly, use the class name in plural form instead to correctly refer to objects (`"movies"` instead of current related names)